src/Sentry/SentryOptionsExtensions.cs

```diff
@@ -387,7 +351,12 @@ public static IEnumerable<ISentryEventExceptionProcessor> GetAllExceptionProcess
     /// <param name="sentryStackTraceFactory">The stack trace factory.</param>
     public static SentryOptions UseStackTraceFactory(this SentryOptions options, ISentryStackTraceFactory sentryStackTraceFactory)
     {
-        options.SentryStackTraceFactory = sentryStackTraceFactory ?? throw new ArgumentNullException(nameof(sentryStackTraceFactory));
+        if (sentryStackTraceFactory is null)
```

Comment:
no functional change here, just style?

Reply:
It's because of successive refactor. I'll fix it.